integration/performance_test.go

```diff
@@ -46,7 +46,7 @@ func BenchmarkIntegration_CustomVotesScenario(b *testing.B) {
 	b.Logf("using temp dir %s", dirPath)
 
 	// ##### CREATE NODES #####
-	nodes := setupDVotingNodes(b, numNodes, dirPath)
+	nodes := setupDVotingNodes(b, numNodes, dirPath,nil)
```

Comment:
I suspect you want to run `go fmt` :-) 

```suggestion
	nodes := setupDVotingNodes(b, numNodes, dirPath, nil)
```